src/hooks/useSettingsUtilities.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  useDeskproAppClient,
+  useDeskproLatestAppContext,
+} from "@deskpro/app-sdk";
+import { IBookmark } from "../types/bookmarks";
+
+export const useSettingsUtilities = () => {
```

Comment:
Please, could you add the types so that it's clear what this hook return

Reply:
The types are inferred from usage.